packages/scripts/src/ManualRelay.js

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
+// This script allows you to manually relay a particular across deposit.
+// Example:
+// NODE_URL_1=https://mainnet.infura.io/v3/SOME_PROJECT_ID NODE_URL_10=https://optimism-mainnet.infura.io/v3/SOME_PROJECT_ID \
+//   ./src/ManualRelay.js --chainId 10 --depositId 100 --network mainnet_mnemonic
+// Optional env overrides:
+//   BRIDGE_ADMIN_ADDRESS
+//   BRIDGE_DEPOSIT_ADDRESS
+//   L2_END_BLOCK_NUMBER
+//
+// Note: if you want to run with GCKMS keys, the command will look like this:
+// NODE_URL_1=https://mainnet.infura.io/v3/SOME_PROJECT_ID NODE_URL_10=https://optimism-mainnet.infura.io/v3/SOME_PROJECT_ID \
+//   GOOGLE_APPLICATION_CREDENTIALS=/your/credentials/file.json ./src/ManualRelay.js --chainId 10 --depositId 100 \
+//   --network mainnet_gckms --keys your_gckms_key_name
+
+const argv = require("minimist")(process.argv.slice(), { string: ["depositId", "chainId"] });
+
+const { getWeb3ByChainId } = require("@uma/common");
+const { InsuredBridgeL1Client, InsuredBridgeL2Client, Logger } = require("@uma/financial-templates-lib");
+const sdk = require("@uma/sdk");
+
+const depositBoxMapping = {
+  10: "0x3baD7AD0728f9917d1Bf08af5782dCbD516cDd96",
+  42161: "0xD8c6dD978a3768F7DDfE3A9aAD2c3Fd75Fa9B6Fd",
+  288: "0xCD43CEa89DF8fE39031C03c24BC24480e942470B",
+};
+
+async function main() {
+  if (!argv.chainId) {
+    throw new Error("You must provide the --chainId argument");
+  } else if (!argv.depositId) {
+    throw new Error("--depositId argument must be provided");
+  }
+
+  const l1Web3 = getWeb3ByChainId(1);
+  const l2Web3 = getWeb3ByChainId(Number(argv.chainId));
+  const [account] = await l1Web3.eth.getAccounts();
+  const latestL2BlockNumber = process.env.L2_END_BLOCK_NUMBER || (await l2Web3.eth.getBlockNumber());
+
+  const l1Client = new InsuredBridgeL1Client(
+    Logger,
+    l1Web3,
+    process.env.BRIDGE_ADMIN_ADDRESS || "0x30B44C676A05F1264d1dE9cC31dB5F2A945186b6",
+    sdk.across.constants.RATE_MODELS
+  );
+  await l1Client.update();
+
+  const l2Client = new InsuredBridgeL2Client(
+    Logger,
+    l2Web3,
+    process.env.BRIDGE_DEPOSIT_ADDRESS || depositBoxMapping[Number(argv.chainId)],
+    Number(argv.chainId),
+    latestL2BlockNumber - 9900,
+    latestL2BlockNumber
+  );
+  await l2Client.update();
+
+  const deposit = l2Client.getAllDeposits().find((deposit) => deposit.depositId === Number(argv.depositId));
+  if (!deposit) throw new Error(`No deposit found for ${argv.depositId}`);
+
+  const txn = l1Client.bridgePools[deposit.l1Token].contract.methods.relayAndSpeedUp(
```

Comment:
Should we add some logs or warning or checking about whether caller has enough funds to post a proposal bond and relay?

Reply:
Won't it just error? Is there a reason it's preferable to manually check for that ahead of time as opposed to just throwing the error?

Reply:
I agree it'll throw an error but perhaps add some more logs before and after the `.call` and `.send` blocks below so the runner of the script knows where it is in the execution.

Reply:
Can do!